src/phylo/branch_basics.jl

```diff
@@ -0,0 +1,47 @@
+# phylo/branch_basics.jl
+# ======================
+#
+# Types and methods for phylogenetic trees.
+#
+# Part of the Bio.Phylo module.
+#
+# This file contains methods for querying and setting attributes on branches,
+# as well as adding and removing branches.
+#
+# This file is a part of BioJulia. License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+
+function branchlength(tree::Phylogeny, edge::Edge)
+    error("No method for branchlength for this kind of phylogeny.")
```

Comment:
Julia will automatically throw `MethodError` if there is no matching method. Why this definition is needed?